### ISSUE TITLE
Bower central has moved and Bower assets no longer download.  Fix being provided in a pull request.

### ISSUE THREAD

**Issue by bkoehm** [craigburke/client-dependencies-gradle#91]
Bower has moved to https://registry.bower.io.  The old URL gives a 308 Permanent Redirect, which url.getText() in BowerRegistry.loadInfoFromRegistry() does not seem to follow.  Instead, it returns either a null or a blank string.  This causes an eventual Gradle exception that may look like this: 
```
* What went wrong:
Execution failed for task ':clientInstall'.
> org.ajoberstar.grgit.exception.GrgitException: Problem cloning repository.
```
This pull https://github.com/craigburke/client-dependencies-gradle/pull/90 has a fix and does the following:
* Changes the Bower URL
* Adds a check after url.getText() to log a warning if nothing was returned so that the user may get a better understanding of what went wrong when the Bower repository is not returning a JSON response.
* Increments the plugin version

**Comment by cknuth** [comment]
FYI while waiting for the pull request to be accepted I've got around this issue defining a custom bower registry in build.gradle like so:

```
    registry 'realBower', type:'bower', url:'https://registry.bower.io'
    realBower {
        ...
    }
```

**Comment by bkoehm** [comment]
@cknuth Thanks.  Yes, I tried this too, but it didn't work for me for some reason.  It seemed like somewhere it was still trying to contact the the old URL.  I'm not sure if it was something in a bower.json file that made it look in the old location or not.  At any rate, I gave up on that and did the patch seen in the pull request.

**Comment by amimas** [comment]
The workaround mentioned by @cknuth worked in my case.
I hope the fix is released soon.

**Comment by craigburke** [comment]
Fixed by #90 which has been released in `1.4.1`
